paying_for_college/templates/worksheet.html

```diff
@@ -2680,93 +2679,111 @@ <h2 class="step_heading">
-                            <h3 class="option_heading">
-                                Make sure you don’t have to take classes twice.
-                            </h3>
-                            <p>
-                                <a href="http://www.consumer.ftc.gov/articles/0395-choosing-college-questions-ask" target="_blank">Ask if credits from your school transfer
-                                </a> to degrees earned at other schools.
-                                <span id="option_completion-rate-content">
-                                <span id="option_completion-rate" data-financial="gradRate"
-                                data-percentage_value="true">[XX]</span>% of
-                                students who start at this school finish here.
-                                </span>
-                                In the case you decide to finish your
-                                education somewhere else, it’s important to
-                                know if the hard work you’ve put into your
-                                degree will be honored at another school.
-                            </p>
-                        </section>
-                        <section class="option option__explore-schools">
-                            <h3 class="option_heading">
-                                Explore other schools that have similar
-                                programs.
-                            </h3>
-                            <p>
-                                Review your total cost of attendance and
-                                determine if a less expensive school option
-                                (such as a community college) might also meet
-                                your educational needs.
-                                <a href="https://collegescorecard.ed.gov/"
-                                class="scorecard-link"
-                                target="_blank">
-                                    See all schools
-                                </a>
-                                that offer the same type of program in your
-                                area.
-                            </p>
-                        </section>
-                        <section class="option option__take-action column-well
-                        column-well__emphasis">
+                        <div class="get-options__dynamic">
+                            <section class="option option__maximize-grants">
+                                <h3 class="option_heading">
+                                    Maximize all available grants and scholarships.
+                                </h3>
+                                <p>
+                                    Consider <a href="https://studentaid.ed.gov/sa/types/grants-scholarships"
+                                    target="_blank"> applying for additional
```

Comment:
Short answer:
Yes, there is a reason. 😄 

Long answer:
The entire tool is stateless, and filled with external links. Opening a link in the same tab would remove any and all edits made as the user went through this rather extensive tool.

We made a decision as a team that the external-links-in-same-window makes sense for static web pages, but we shouldn't follow that guideline for dynamic tools such as this one. The decision was shared at multiple design working sessions dating back to late last year, as well as at the D&D wide Friday design presentations. @ascott1 was on the team at the time the decision was made.

The plan is to call this out formally on the Design Manual repo as undesirable behavior for tools, using examples from this and Owning a Home (and possible guidance given on the Digital Service listserve from elsewhere in government). It hasn't been done yet because it's lower in priority for us than getting the project out, and because we thought we shared the knowledge internally pretty well as all our internal review meetings. 😕  Guess not.

I'll add the link guidance revision request to our Design Manual task board today.